src/Info/Info.js

```diff
@@ -1,13 +1,14 @@
 import React, { Component } from "react"
 import './Info.css'
 
-const Info = (props) => {
+const Info = ({ movie, showAll }) => {
   return (
     <div className='info'>
       <h3>Movie Details Here</h3>
-      <h3>Title: {props.movie.movie.movie.title}</h3>
-      <h3>Average Rating: {props.movie.movie.movie.average_rating.toFixed(1)}</h3>
-      <h3>Release Date: {props.movie.movie.movie.release_date}</h3>
+      <h3>Title: {movie.title}</h3>
+      <h3>Average Rating: {movie.average_rating}</h3>
+      <h3>Release Date: {movie.release_date}</h3>
+      <button className='return-button' onClick={showAll}>Return</button>
```

Comment:
When I see a lot of "movie.something", it can indicate an opportunity to use destructuring to avoid that repetitive need for the dot notation syntax.  Not a directive, just something to consider. You're the dev, you decide :)